### ISSUE TITLE
[Quest] Escape from Coliskar Cistern

### ISSUE THREAD

**Issue by TheJesterrr** [chromiecraft/chromiecraft#5148]
### What client do you play on?

enUS

### Faction

Alliance

### Content Phase:

70

### Current Behaviour

Upon accepting the quest 'Escape from Coliskar Cistern' in SMV, the quest will immediately fail and the escort NPC will disappear. 

### Expected Blizzlike Behaviour

Escort quest through the Coliskar Cave occurs.

### Source

https://www.wowhead.com/tbc/quest=10451/escape-from-coilskar-cistern

### Steps to reproduce the problem

1. Enter Coliskar Cave.
2. Kill nearby enemy to the NPC quest giver to 'activate' the quest giver.
3. Accept quest
4. It will fail within ~3 seconds.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/0200a288cd89034a8b491253006a0b0055178deb

### Operating system

WIndows 10

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by nJDOA** [comment]
Confirmed. Failes instantly and the Earthmender disappears.

**Comment by Stump876** [comment]
Sorry I just searched the NPC and wrote up an entire report before I realized this was here. https://github.com/chromiecraft/chromiecraft/issues/5172 This is valid

**Comment by Peppersprayz** [comment]
It's still bugged today despite the "fix"

**Comment by TheJesterrr** [comment]
Confirming it is still bugged. I am getting an instant fail upon accepting the quest and the NPC disappears.

**Comment by Depe79** [comment]
I can confirm it's still bugged.

**Comment by Gultask** [comment]
I don't understand why this is happening. I've never managed to reproduce the issue myself, it always worked for me.

**Comment by Stump876** [comment]
I see the screen shots and I believe it, but bs far as I can tell it's working.  But, I did manage to break it after some time. 

She is getting hit with something, and it's causing her to enter combat- popping the bubble and dropping her down. After that, when the keeper is killed she will walk to the quest start point as normal, but will not move even if attacked. What exactly is hitting her is still unknown to me, I was using a GM spell. I'd have to go through every class and every spell. And I'm not even sure that's the answer. However, once she died everything went back to normal. So IDK. 

Again, if anyone has any additional info, a class, a spell, anthing that may be causing it, feel free to post it here.

**Comment by TheJesterrr** [comment]
If it helps, made a couple recordings of the fails (Off video I tried solo). With these two, one is in a party with the pally, and the other is outside of a party spectating the pally try it solo himself. We both get the same instant fail.

[https://www.youtube.com/watch?v=t7UnV6Qrc4I](https://www.youtube.com/watch?v=t7UnV6Qrc4I) - short 14 second video spectating with combat log open.

[https://youtu.be/J2rL1IlpW0s](https://youtu.be/J2rL1IlpW0s) - 1:30min clip showcasing the kill, freeing, and fail. Complete with commentary... sort of.

Three fails back-to-back-to-back. The NPC could not be killed, just disappears. Each new attempt yielded the same result.

**Comment by Stump876** [comment]
> If it helps, made a couple recordings of the fails (Off video I tried solo). With these two, one is in a party with the pally, and the other is outside of a party spectating the pally try it solo himself. We both get the same instant fail.
> 
> https://www.youtube.com/watch?v=t7UnV6Qrc4I - short 14 second video spectating with combat log open.
> 
> https://youtu.be/J2rL1IlpW0s - 1:30min clip showcasing the kill, freeing, and fail. Complete with commentary... sort of.
> 
> Three fails back-to-back-to-back. The NPC could not be killed, just disappears. Each new attempt yielded the same result.

Thanks, I'll  dig into this some more this weekend.

**Comment by Gultask** [comment]
A video helps a lot, thank you!

**Comment by Stump876** [comment]
While I can duplicate this on CC, I can't on my local, and I can't on the PTR. 

Update: After the server reset on CC I was able to complete it with no issue. 

I did see this clip through the ceiling at one point lol
![image](https://user-images.githubusercontent.com/68164076/229030225-4cdf157a-d7ef-4252-916f-9bcfe0922b8b.png)

It actually attacked us when I was doing it on CC

**Comment by Gultask** [comment]
Yeah I meant to fix the pathing for that one way back. She also should have some special behaviour where she plops down an egg every now and then :P

**Comment by Gultask** [comment]
> After the server reset on CC I was able to complete it with no issue.
Tearing my hair out 😤

**Comment by Samsequel** [comment]
Replicated the issue, if the keeper is dead when the npc spawns, it will fail the quest.

**Comment by TheJesterrr** [comment]
Tested 4/6/2023 - Instant Fail remains :( https://www.youtube.com/watch?v=VnfGDQ6zhGY

In good news though with Arca opening I was able to get the SMV quests achievement and no longer need this one!

**Comment by Gultask** [comment]
![image](https://user-images.githubusercontent.com/100873791/230480711-f94bbd24-7370-45b5-95bd-103681135d74.png)

**Comment by 0mnimodus** [comment]
Confirmed. Failes instantly.

**Comment by XC-ahhaVA** [comment]
Still fails instantly. 28/03/2024
![image](https://github.com/chromiecraft/chromiecraft/assets/164023764/8739bbba-8d39-47d5-9298-67359faccc3e)

**Comment by Kosiak** [comment]
Still failing as of 16/09/2024. Tried different variations and no matter what you do u get a fail.

**Comment by Mervin31** [comment]
One try, immediate fail of the quest and NPC disappeared.
![image](https://github.com/user-attachments/assets/8de25a8f-a022-4b35-96ec-ec965164e7da)

**Comment by Mervin31** [comment]
Tried today as well. Another weird thing was mobs in the wall. When jailor dies questgiver flies to the point where she gives the q, the flying is done through the air with the swimming motion. After taking the q there is a sentence and then she disappears and q is failed immediately.
![Screenshot_2](https://github.com/user-attachments/assets/1ff2dd54-4282-4574-90c5-40d78d096d30)
![Screenshot_4](https://github.com/user-attachments/assets/fa906d9a-5aca-438c-8dd1-35634cf7e56a)
![Screenshot_5](https://github.com/user-attachments/assets/ba9fe1c4-c2dc-4117-bf5d-6cdca8fd988d)

**Comment by Mervin31** [comment]
Does this issue have any visibility at all?

**Comment by Annamaria-CC** [comment]
It appears to not have been properly linked previously.  I will fix this mistake.

**Comment by Mervin31** [comment]
npc still flies with swimming and quest still insta fails
![image](https://github.com/user-attachments/assets/49a33963-6d50-4f27-9db4-0b9d3c12be3e)
